website/content/en/docs/Config/Port/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Port Forwarding
+weight: 50
+---
+
+Lima supports automatic port-forwarding of localhost ports from guest to host.
+
+## Port forwarding types
+
+### Using SSH
+
+SSH based port forwarding is the default and current model that is supported in lima.
+
+To use SSH forwarding use the below command
+
+```bash
+LIMA_SSH_PORT_FORWARDER=true limactl start
+```
+
+#### Caveats
+
+- Doesn't support UDP based port forwarding
+- Spans child process on host for running SSH master.
+
+### Using GRPC (Default)
+
+> **Warning**
+> This mode is experimental
```

Comment:
Does this apply to TCP too?

Reply:
Yes correct. The GRPC tunnel itself new right so it applies for both tcp and usp